app/services/invite_participants.rb

```diff
@@ -1,30 +1,33 @@
 # frozen_string_literal: true
 
 class InviteParticipants
-  def run(emails)
+  def self.run(emails)
     logger.info "Emailing Participants"
 
     user = nil
```

Comment:
It was already in but why is user assigned nil and then assigned again on line 9?